bitbots_vision/src/bitbots_vision/vision_modules/color.py

```diff
@@ -94,18 +98,21 @@ def pixel_bgr2hsv(pixel):
 
 
 class HsvSpaceColorDetector(ColorDetector):
+    """
+    HsvSpaceColorDetector is a ColorDetector, that is based on the HSV-color space.
+    The HSV-color space is adjustable by setting min- and max-values for hue, saturation and value.
+    """
     def __init__(self, debug_printer, min_vals, max_vals):
         # type: (DebugPrinter, tuple[int, int, int], tuple[int, int, int]) -> None
         """
-        HsvSpaceColorDetector is a ColorDetector, that is based on the HSV-color space.
-        The HSV-color space is adjustable by setting min- and max-values for hue, saturation and value.
+        Initiating of HsvSpaceColorDetector.
```

Comment:
Initialization of...

Reply:
Closed with 7398d9f2ebe6647e5caa15e9846bfb3e33624e37